File: mgm-gcp/src/main/java/uk/co/cerihughes/mgm/translate/AlbumTranslation.java
package uk.co.cerihughes.mgm.translate;

import org.openapitools.model.AlbumApiModel;
import uk.co.cerihughes.mgm.model.interim.InterimAlbum;

public interface AlbumTranslation extends EntityTranslation {
    AlbumApiModel translate(InterimAlbum interimAlbum);
}

File: mgm-gcp/src/main/java/uk/co/cerihughes/mgm/translate/AlbumTranslation.kt
package uk.co.cerihughes.mgm.translate

import uk.co.cerihughes.mgm.model.interim.InterimAlbum
import uk.co.cerihughes.mgm.model.output.AlbumApiModel

interface AlbumTranslation : EntityTranslation {
    fun translate(interimAlbum: InterimAlbum): AlbumApiModel?
}
